react/components/Lists/index.tsx

```diff
@@ -85,52 +79,50 @@ class Lists extends Component<ListsProps, ListsState> {
     } = this.state
     const { onClose, intl } = this.props
 
-    return !show
-      ? null
-      : createPortal(
+    return createPortal(
+      <Screen>
+        <Header
+          title={intl.formatMessage(messages.myLists)}
+          onClose={onClose}
+          action={() => this.setState({ showCreateList: true })}
+        />
+        {this.renderContent()}
+        {showCreateList && (
+          <div className="fixed vw-100 top-0 bg-base">
+            <CreateList
+              onClose={() => this.setState({ showCreateList: false })}
+              onFinishAdding={this.handleListCreated}
+            />
+          </div>
+        )}
+        {showUpdateList && (
           <Screen>
-            <Header
-              title={intl.formatMessage(messages.myLists)}
-              onClose={onClose}
-              action={() => this.setState({ showCreateList: true })}
+            <UpdateList
+              onClose={() => this.setState({ showUpdateList: false })}
+              list={lists && lists[listSelected]}
+              onFinishUpdate={this.handleListUpdated}
             />
-            {this.renderContent()}
-            {showCreateList && (
-              <div className="fixed vw-100 top-0 bg-base">
-                <CreateList
-                  onClose={() => this.setState({ showCreateList: false })}
-                  onFinishAdding={this.handleListCreated}
-                />
-              </div>
-            )}
-            {showUpdateList && (
-              <Screen>
-                <UpdateList
-                  onClose={() => this.setState({ showUpdateList: false })}
-                  list={lists[listSelected]}
-                  onFinishUpdate={this.handleListUpdated}
-                />
-              </Screen>
-            )}
-            {showListDetails && (
-              <div className="fixed vw-100 top-0 left-0 bg-base">
-                <ListDetails
-                  onClose={() => this.setState({ showListDetails: false })}
-                  listId={lists[listSelected].id}
-                  onDeleted={this.handleDeleteList}
-                />
-              </div>
-            )}
-          </Screen>,
-          document.body
-        )
+          </Screen>
+        )}
+        {showListDetails && (
+          <div className="fixed vw-100 top-0 left-0 bg-base">
+            <ListDetails
+              onClose={() => this.setState({ showListDetails: false })}
+              listId={lists && lists[listSelected].id}
+              onDeleted={this.handleDeleteList}
+            />
+          </div>
+        )}
+      </Screen>,
+      document.body
+    )
   }
 
   private renderLists = (): ReactNode => {
     const { lists } = this.state
     return (
       <Fragment>
-        {lists.length ? (
+        {lists && lists.length ? (
```

Comment:
You should use `.length > 0` or `isEmpty` function, I believe it will render 0 when the list length is 0